products/compute/ansible_version_added.yaml

```diff
@@ -201,6 +201,8 @@
         :version_added: '2.6'
       :kmsKeyName:
         :version_added: '2.6'
+    :physicalBlockSizeBytes:
+      :version_added: '2.8'
```

Comment:
I was not sure how to check the version so just assumed 2.8. If different please let me know how to check the next value.

Reply:
`2.8` is correct. I believe this bit is also generated now if it's not added in the original PR.